components/raftstore-v2/src/operation/command/admin/mod.rs

```diff
@@ -110,6 +111,10 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
                     }
                 }
                 AdminCmdType::CompactLog => self.propose_compact_log(ctx, req),
+                AdminCmdType::UpdateGcPeer => {
```

Comment:
If I understand it correctly, we rely on this cmd to propagate the removed peer list. 
But what if a conf change happen and a newly added node become leader. And at that time how does the new node get the removed node list? (maybe this cmd not run yet before that node get elected).
My original thought is this information needs to be transferred from the snapshot.

Reply:
The list is part of region local state and snapshot state. Every node should have the exact same peer list when they applies to the same index.

Reply:
The protocol updates can be found in https://github.com/pingcap/kvproto/pull/1040.

Reply:
> The protocol updates can be found in [pingcap/kvproto#1040](https://github.com/pingcap/kvproto/pull/1040).

I see. So it's part of snapshot. Thanks.